packages/types/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
 ## [Unreleased]
+## [1.3.3-alpha.1](https://github.com/cennznet/api.js/compare/release/1.3.2...prerelease/1.3.3-alpha.1) (18/02/2021)
+### Added:
+    - Added RPC calls for staking(accruedPayout) and grandpa(restartVoter)
```

Comment:
Can you add a note that the sylo extrinsics were removed in v38 runtime which is reflected here in API